modal/volume.py

```diff
@@ -419,6 +419,27 @@ async def copy_files(self, src_paths: Sequence[Union[str, bytes]], dst_path: Uni
         """
         Copy files within the volume from src_paths to dst_path.
         The semantics of the copy operation follow those of the UNIX cp command.
+
+        The `src_paths` parameter is a list. If you want to copy a single file, you should pass a list with a
+        single element.
+
+        `src_paths` and `dst_path` should refer to the desired location *inside* the volume. You do not need to prepend
+        the volume mount path.
+
+        **Usage**
+
+        ```python notest
+        @stub.function(volumes={"/root/foo": volume})
+        def g():
+            volume.reload()  # Fetch latest changes
+            with open("/root/foo/bar/example.txt", "r") as f:
+                print(f.read())
+
+            volume.copy_files(["bar/example.txt"], "bar2")  # Copy files to another directory
+            volume.copy_files(
+                ["bar/example.txt"], "bar/example2.txt"
+            )  # Rename a file by copying
```

Comment:
I think it's an anti-pattern to use the Volume API methods when the volume is mounted to the function; we should probably demonstrate usage in a non-modal-app context (and maybe say that explicitly).

Reply:
yup, we had a separate discussion about this, updated :-)